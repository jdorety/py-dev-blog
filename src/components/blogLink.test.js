import React from "react"
import renderer from "react-test-renderer"

import BlogLink from "./blogLink"

it("renders correctly", () => {
  const tree = renderer
    .create(
      <BlogLink title="Test title" path="/test/test.md">
        BlogLink
      </BlogLink>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
