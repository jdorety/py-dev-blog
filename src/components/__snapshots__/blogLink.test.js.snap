// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  href="/test/test.md"
>
  <div
    className="blog-link"
  >
    <h3>
      Test title
    </h3>
  </div>
</a>
`;
